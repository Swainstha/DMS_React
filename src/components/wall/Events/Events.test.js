import React from 'react';
import Events from './index';
import testSnapFunction from '../../../util/testSnapFunction';
const dummy_data= {
    eventList:[{id:3},{id:1},{id:2},{id:4}]
}

testSnapFunction("<Events/>", "Snapshot test for PhroductGridItem",<Events {...dummy_data}/> )

