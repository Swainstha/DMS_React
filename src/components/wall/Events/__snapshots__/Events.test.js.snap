// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Events/> Snapshot test for PhroductGridItem 1`] = `ShallowWrapper {}`;
